live-build/misc/migration-scripts/dx_apply

```diff
@@ -106,13 +106,8 @@ RPOOL=${RDS%%/*}
 #
 # Cleanup any previous intermediate state.
 #
-rm -rf /tmp/delphix.* ||
-	die "failed to destroy old delphix temporary directories"
-rm -f /boot/vmlinuz-* /boot/initrd.img-* ||
-	die "failed to destroy previously copied Linux kernel data"
-zfs destroy -r "$RPOOL/ROOT" 2>/dev/null
-zfs list "$RPOOL/ROOT" 2>/dev/null &&
-	die "could not destroy linux root dataset from previous run"
+"${BASH_SOURCE%/*}/dx_delete" ||
```

Comment:
dx_delete only appears to destroy rpool/ROOT, and nothing else. We didn't need to do the other steps that were removed?

Reply:
Which version of dx_delete are you looking at? I updated it recently to clean things up more comprehensively.

Reply:
I've updated the description to explain this.

Reply:
@shartse ha! I had a super outdated version of appliance-build that I was looking at. Indeed, fetching the latest one makes everything clear to me. Thanks for pointing this out. I even reviewed your code (<embarrassed>)...!